### PR TITLE
Add customers path

### PR DESCRIPTION
**Pull request by chandlercarlile**
Issue: https://github.com/slingshotinc/web-app/issues/4194

This adds documentation for the bulk operations customers endpoint. @slingshot-roybabbel and I worked together on #1, so this PR is pretty much a copy-pasta of that PR applied to the customer endpoint.

### CODE REVIEW

File: openapi/paths/v1/bulk_operations/notification_settings.yaml [hunk 1]
```diff
@@ -10,7 +10,7 @@ put:
     - basic_auth: []
   responses:
     '200':
-      $ref: ../../../components/responses/bulk_operations/post/success.yaml
+      $ref: ../../../components/responses/bulk_operations/put/success.yaml
```
Comment (chandlercarlile) [inline openapi/paths/v1/bulk_operations/notification_settings.yaml:13]: This success response is the exact same for both `bulk_operations` endpoints. Curious your thoughts @slingshot-roybabbel  @Rockster160  about if we should technically have a separate ref for the `customers` vs the `notification_settings` despite it being the same?

Possibly even just a "shared component" that is used by `/components/responses/bulk_operations/notification_settings/put/success.yaml` and `/components/responses/bulk_operations/customers/put/success.yaml`

File: openapi/paths/v1/bulk_operations/customers.yaml [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+  operationId: createOrUpdateCustomers
+  parameters:
+    $ref: ../../../components/parameters/CompanyId.yaml
+  security:
+    - basic_auth: []
+  responses:
+    '200':
+      $ref: ../../../components/responses/bulk_operations/put/success.yaml
+    '400':
+      description: Invalid csv provided
```
Comment (kellyjensen) [inline openapi/paths/v1/bulk_operations/customers.yaml:15]: Should upcase `csv` to be consistent.

File: openapi/paths/v1/bulk_operations/notification_settings.yaml [hunk 3]
```diff
@@ -10,7 +10,7 @@ put:
...
     '400':
       description: Invalid csv provided
```
Comment (kellyjensen) [inline openapi/paths/v1/bulk_operations/notification_settings.yaml:15]: Upcase `csv`

File: openapi/paths/v1/bulk_operations/customers.yaml (hunk 2 above)
Comment (chandlercarlile) [reply openapi/paths/v1/bulk_operations/customers.yaml:15]: Ah good catch.

File: openapi/paths/v1/bulk_operations/notification_settings.yaml (hunk 1 above)
Comment (Rockster160) [reply openapi/paths/v1/bulk_operations/notification_settings.yaml:13]: I'm fine either way. I always like to DRY up code- but don't want to make it overly complex if anybody tries to extend or modify the functionality.

**Review by Rockster160 — APPROVED** [review]

File: openapi/paths/v1/bulk_operations/notification_settings.yaml (hunk 1 above)
Comment (chandlercarlile) [reply openapi/paths/v1/bulk_operations/notification_settings.yaml:13]: I was working on another PR for doc stuff, and actually decided to come back to this one and split it out. So I'll do that before finishing here.

**Review by kellyjensen — APPROVED** [review]

**Pull request merged**